### PR TITLE
Update Map

### LINKED ISSUE #153: Revise Homepage Counts
From today's "PIF Website Changes" discussion:

  + Remove all counts except for "122 Fellows" and "28 Agencies"
  + Revise region counts (investigate whether or not this happens automatically, and perhaps spin-off a new issue to automate the updating of these numbers)

screen shot 2017-12-19 at 5 19 06 pm

### PR DESCRIPTION
**Pull request by s2t2**
Resolves #153 

Preview: https://federalist-proxy.app.cloud.gov/preview/s2t2/pif-website/mjr-counts/

Additions:

  + Provides instructions on how to update the map
  + Provides a development option for logging counts of agencies, fellows, and fellows by region
  + Provides a development option for validating regions
  + Fixes regions for two fellows
  + Includes separate region images (not being used at the moment, but they might be useful in the future if someone wants to automate the image)

Revisions:

  + Moves and renames map assets
  + Updates map and page content to reflect current fellow, region, and agency counts 

Questions/Concerns:

  + In the previous map the number of International/Outside fellows was 3, but based on the data, this is 2. Not sure who the third was previously.

### CODE REVIEW

**Comment by s2t2** [comment]
Updated counts: 

```rb
{:total=>122,
 :by_region=>
  {:northeast=>41, :west=>45, :midwest=>10, :south=>24, :outside=>2}}
```

**Comment by s2t2** [comment]
Updated map preview:

screen shot 2018-01-24 at 3 46 36 pm
screen shot 2018-01-24 at 3 47 07 pm

**Comment by s2t2** [comment]
Running into some federalist build issues, most likely due to requiring `activesupport`. Investigating...

**Comment by s2t2** [comment]
The federalist build is still breaking. Not sure if there is any way to use `activesupport`, even if it is included in a non-development group in the Gemfile. Not confident there is a solution, so I'm going to remove this dependency (it was non-critical).

**Comment by s2t2** [comment]
The federalist issue was unrelated, due to recent changes in cloud.gov, and has since been resolved.

File: index.html [hunk 1]
```diff
@@ -50,18 +50,31 @@ <h2>Recruiting industry leaders to serve as change agents</h2>
...
+            -->
+            <!-- Updating this section Step 2:
+              ... in the PIF shared google drive under Design > Visual Design Assets,
+              ... find the map source file named something like "PIF-regions-map-YYYYMMDD.key",
+              ... then open the file, and edit each of the numbers,
+              ... and recognize each of the circles is really a pie chart,
+              ... and double-click on each of the pie charts to "Edit Chart Data",
+              ... then enter in the updated total number of PIFs, and the updated number of PIFs for that region,
+              ... then save a new version of the keynote file to google drive,
+              ... then take a screenshot of the final map and save to this repo as "assets/images/map/pif-map-YYYYMMDD.png",
```
Comment (zachgoldfine44) [inline index.html]: It could help to clarify to what repo you refer by "this repo"

Comment (zachgoldfine44) [reply index.html]: Although I guess you specify the file name requested right after, so you can probably ignore my comment.

**Comment by s2t2** [comment]
@zachgoldfine44 I changed to "save to this pif-website repo". If this is good to merge, give me the thumbs up.

**Pull request merged**